includes/GbWeiss.php

```diff
@@ -48,7 +51,7 @@ final class GbWeiss extends Singleton
     private $orderStateRepository;
 
     /**
-     * Option Page Slug
+     * Initialize GbWeiss Plugin
```

Comment:
I think we should keep the old comment here.

```suggestion
     * Option Page Slug
```